routes/v1.js

```diff
@@ -88,6 +89,7 @@ function addRoutes(app, peliasConfig) {
       postProc.parseBoundingBox(),
       postProc.normalizeParentIds(),
       postProc.geocodeJSON(peliasConfig, base),
+      postProc.assignLabels(require('pelias-labels')),
```

Comment:
I think it makes more sense to generate labels before geojsonifying the results. It feels like the geojsonify step is like a final packaging up of the data, so adding properties afterwards could lead to those property not being packaged properly. Thoughts?